src/client/src/apps/MainRoute/widgets/workspace/selectors.ts

```diff
@@ -31,17 +32,25 @@ const makeExternalWindowProps: (key: string, tileLayout?: WindowPosition) => Ext
 
 const getSpotTiles = (state: GlobalState) => state.currencyPairs
 const getSpotTilesLayout = (state: GlobalState) => state.layoutService.spotTiles
+const getSpotTilesData = (state: GlobalState) => state.spotTilesData
+
+const spotTilesDataTypeChecker = (spotTilesData: SpotTileState, key: string) => typeof spotTilesData !== 'undefined' && typeof spotTilesData[key] !== 'undefined'
```

Comment:
Pretty sure `spotTilesData` is always an object, so you can reduce this to only `typeof spotTilesData[key] !== 'undefined'`.

If that is the case, then do you need this function? See comment below.